styles.css

```diff
@@ -200,8 +201,10 @@ button {
   color: #6070ff;
   font-family: 'Poppins', sans-serif;
   font-weight: 500;
-  size: 12px;
-  list-style: 16px;
+
+  /* size: 12px; */
+
+  /* list-style: 16px; */
```

Comment:
- [OPTIONAL] Please it is against the best practices to leave un-used code as a comment in your code base. It makes your code look unprofessional. please remove this. :smile: :+1: